contracts/sol6/Dao/KyberDAO.sol

```diff
@@ -775,7 +770,7 @@ contract KyberDAO is IKyberDAO, EpochUtils, ReentrancyGuard, Utils5, DaoOperator
     /**
      * @dev options are indexed from 1
      */
-    function validateVoteOption(uint256 campaignID, uint256 option) internal view returns (bool) {
+    function validateVoteOption(uint256 campaignID, uint256 option) internal view {
```

Comment:
I already had this in my cleanup dao PR: https://github.com/KyberNetwork/smart-contracts/pull/972/files
Should we close this PR?

Reply:
I want to squash this PR with other PR for removing pattern of require true.
And both will reference a specific comment by chain security.